README.md

```diff
@@ -10,4 +11,14 @@ function mystery(a) {
     if(foo > a[0]) return foo;
     else return a[0];
 }
+
+/*
+The above function takes an array 'a' and returns the maximum number in the array
```

Comment:
What if the elements are not numbers?

Reply:
if the elements are not numbers javascript compares the strings in lexicographical order. Meaning the elements are compared character by character accordingly and the greatest one will be returned. For example, if we compare ABC, ABE the maximum will be ABE. Likewise is we compare A5B, A6B  the maximum will be A6B.

Reference: https://javascript.info/comparison#:~:text=String%20comparison,compared%20letter%2Dby%2Dletter.

Reply:
Ok, can you clarify that in the description please?

Reply:
I updated the description as requested.